docs/administration-guide/aggregation/ingest-example.md

```diff
@@ -0,0 +1,607 @@
+                "vertex": "id.person.string"
+            },
+            "Repository": {
+                "description": "Entity representing a repository vertex",
+                "vertex": "id.repo.string"
+            }
+        }
+    }
+    ```
+
+=== "types.json"
+
+    ```json
+    {
+        "types": {
+            "id.person.string": {
+                "description": "A basic type to hold the string id of a person entity",
+                "class": "java.lang.String"
+            },
+            "id.repo.string": {
+                "description": "A basic type to hold the string id of a repository entity",
+                "class": "java.lang.String"
+            },
+            "property.integer": {
+                "description": "A basic type to hold integer properties of elements",
+                "class": "java.lang.Integer",
+                "aggregateFunction": {
+                    "class": "uk.gov.gchq.koryphe.impl.binaryoperator.Sum"
+                }
+            },
+            "true": {
+                "description": "A simple boolean that must always be true.",
+                "class": "java.lang.Boolean",
+                "validateFunctions": [
+                    {
+                        "class": "uk.gov.gchq.koryphe.impl.predicate.IsTrue"
+                    }
+                ]
+            }
+        }
+    }
+    ```
+
+In the above schema you can see we have applied an aggregation function to the
+`"property.integer"` type which will sum the property to give a total. For this
+function we must specify a class that will do the aggregation. There exists a
+few default classes and some additional ones implemented by the Koryphe module
+which you can read more about in the [reference guide](../../reference/binary-operators-guide/binary-operators.md).
+
+!!! tip
+    It is possible to create your own aggregation functions however, they must
+    extend the [`java.util.function.BiFunction`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/function/BiFunction.html?is-external=true)
+    interface.
+
+Loading the data into a Graph using the example schema we can form the Graph
+and see the aggregation in action. First loading the data via the REST API
+using the `AddElements` operation like below:
+
+```json
+{
+    "class": "AddElements",
+    "input": [
+        {
+            "class": "Edge",
+            "group": "Commit",
+            "source": "Dave",
+            "destination": "r1",
+            "directed": true,
+            "properties": {
+                "added": 6,
+                "removed": 8
+            }
+        },
+        {
+            "class": "Edge",
+            "group": "Commit",
+            "source": "Dave",
+            "destination": "r1",
+            "directed": true,
+            "properties": {
+                "added": 35,
+                "removed": 10
+            }
+        },
+        {
+            "class": "Entity",
```

Comment:
You actually don't need to add entities in this example, Gaffer will let you just add edges, but not sure if that would make it clearer or more confusing.

Reply:
True I thought would be bit clearer if they were added in so the vertexes are grouped like the mermaid diagram